src/manifolds/DirectSumBundle.jl

```diff
@@ -0,0 +1,232 @@
+
```

Comment:
This is probably a bit WIP still, but again the functions should be documented.

Reply:
I've decided to leave it for a future PR.